livekit-android-sdk/src/main/java/io/livekit/android/room/RTCEngine.kt

```diff
@@ -1028,6 +1028,20 @@ internal constructor(
                 listener?.onTranscriptionReceived(dp.transcription)
             }
 
+            LivekitModels.DataPacket.ValueCase.METRICS -> {
+                // TODO
+            }
+
+            LivekitModels.DataPacket.ValueCase.CHAT_MESSAGE -> {
+                // TODO
+            }
+
+            LivekitModels.DataPacket.ValueCase.RPC_REQUEST,
+            LivekitModels.DataPacket.ValueCase.RPC_ACK,
+            LivekitModels.DataPacket.ValueCase.RPC_RESPONSE,
+            -> {
+                // TODO
```

Comment:
there's nothing to actually do, right?

Reply:
Nothing to do in this PR, but there is client work required in the future for these data messages types right?